test/serializer/additional-functions/NestedOptimizedFunction1.js

```diff
@@ -0,0 +1,14 @@
+// does not contain:1 + 2
+// skip this test for now
+// (It fails because 1 + 2 not (yet) getting optimized, as nested calls to __optimize seem to just get ignored?)
```

Comment:
We don't do a fixpoint when evaluating registered functions to optimize, we just take whatever was regestered at the end of global code and evaluate that.